<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputWidget</class>
 <widget class="QWidget" name="OutputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="infoToolButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="sate_resources.qrc">
         <normaloff>:/img/icons/info.png</normaloff>:/img/icons/info.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="errorToolButton">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="sate_resources.qrc">
         <normaloff>:/img/icons/error.png</normaloff>:/img/icons/error.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="jsonToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="sate_resources.qrc">
         <normaloff>:/img/icons/bug.png</normaloff>:/img/icons/bug.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="entitiesTtoolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="sate_resources.qrc">
         <normaloff>:/img/icons/entity_orange.png</normaloff>:/img/icons/entity_orange.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="messagesToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="sate_resources.qrc">
         <normaloff>:/img/icons/message_orange.png</normaloff>:/img/icons/message_orange.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="requestToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="sate_resources.qrc">
         <normaloff>:/img/icons/gear_orange.png</normaloff>:/img/icons/gear_orange.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="responseToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="sate_resources.qrc">
         <normaloff>:/img/icons/response_orange.png</normaloff>:/img/icons/response_orange.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="output">
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="sate_resources.qrc"/>
 </resources>
 <connections/>
</ui>
